defining_policies_and_profiles/topics/editing_policy_event_assignments.adoc

```diff
@@ -8,7 +8,7 @@ The policy evaluates its scopes and conditions when specified events occur in yo
 
 . Click image:../images/1847.png[image] (*Configuration*), image:../images/1880.png[image] (*Edit this Policy's Event assignments*).
 
-. Check all the events you want to assign to this policy. For a description of the events, see link:https://access.redhat.com/documentation/en/red-hat-{product-title}/version-4.1-beta/defining-policies-and-profiles/#chap_events[Events].
+. Check all the events you want to assign to this policy. For a description of the events, see link:https://access.redhat.com/documentation/en/{product-title}-{product-title}/version-{product-version}-beta/defining-policies-and-profiles/#chap_events[Events].
```

Comment:
Yeah, it makes sense.

What are the links we should make the conditional statements print?

@CarlosGlez7 @katafira @sergio-ocon @delaluzparra